README.md

```diff
@@ -66,6 +66,7 @@ With extra option you can provide additional data about the NFT (in **item_info*
     author: String
     image_url: String
     name: String
+    category: String
```

Comment:
Just a thought - do we explain somewhere what category means, maybe in the main docs? Because right now it doesn't really speak for itself